lighthouse-cli/test/smokehouse/test-definitions/screenshot.js

```diff
@@ -54,9 +54,13 @@ const expectations = {
         // Note: The first number (5) in these ids comes from an executionContextId, and has the potential to change.
         // The following P is the same element as above but from a different JS context. This element
         // starts with height ~18 and grows over time. See screenshot.html.
-        '5-0-BODY': {_legacyOnly: true, ...elements.body},
-        '5-2-P': {_legacyOnly: true, ...elements.p},
-        '5-3-HTML': {_legacyOnly: true},
+        '5-0-BODY': {_legacyOnly: true, ...elements.body, _maxChromiumVersion: '104.0.5098.0'},
+        '5-2-P': {_legacyOnly: true, ...elements.p, _maxChromiumVersion: '104.0.5098.0'},
+        '5-3-HTML': {_legacyOnly: true, _maxChromiumVersion: '104.0.5098.0'},
+        // Legacy runner execution context ID changed after 104.0.5100.0
```

Comment:
can you add a link to your bisect result

Reply:
also this is the version https://chromium.googlesource.com/chromium/src/+/5150754880affddb3836d38db5ca30a0c53a70c0/chrome/VERSION